its/src/test/java/com/sonar/it/csharp/CoverageTest.java

```diff
@@ -202,6 +217,10 @@ private BuildResult analyzeMultipleProjectsTestProject(String coverageProperty,
     return Tests.analyzeProject(temp, "CoverageTest.MultipleProjects", "no_rule", coverageProperty, coverageFileName);
   }
 
+  private BuildResult analyzeMultipleFrameworksTestProject(String coverageProperty, String coverageFileNames) throws IOException {
+    return Tests.analyzeProject(temp, "CoverageTest.MultipleFrameworks", "no_rule", coverageProperty, coverageFileNames);
```

Comment:
Why is this a separate private method? It's called just once

Reply:
For consistency with `analyzeCoverageTestProject` and `analyzeMultipleProjectsTestProject`.